src/views/virtualmachines/details/tabs/configuration/details/DetailsTab.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+
+import { V1VirtualMachine, V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import SidebarEditor from '@kubevirt-utils/components/SidebarEditor/SidebarEditor';
+import { PATHS_TO_HIGHLIGHT } from '@kubevirt-utils/resources/vm/utils/constants';
+import { PageSection } from '@patternfly/react-core';
+
+import { onSubmitYAML } from './utils/utils';
+import DetailsSection from './DetailsSection';
+
+type DetailsTabProps = RouteComponentProps<{
+  name: string;
+  ns: string;
+}> & {
+  obj?: V1VirtualMachine;
+  vmi?: V1VirtualMachineInstance;
+};
```

Comment:
I think for this you can create a type in the VirtualMachineConfigurationTab,export it and import it here and in all the Tab components inside ConfigurationTab

Reply:
yes correct, it repeats a lot, I created a type all can consume.

it won't be changing a lot and if will is changed it will pass the change to all inner tabs so 1 location for this prop kind is good.